src/pages/index.js

```diff
@@ -101,10 +101,6 @@ profileImageButton.addEventListener("click", () => {
   profileImageModal.open();
 });
 
-profileImageSaveButton.addEventListener("click", () => {
-  handleProfileImageForm();
-});
-
 function processLikeClick(card) {
   const isLiked = card.isLiked();
   console.log(isLiked);
```

Comment:
```js

    api.removeCardLike(card._cardId).then((data) => {
      card.setLikesInfo(data.likes);
    });
```


You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**